easybuild/framework/easyconfig/tools.py

```diff
@@ -55,34 +55,28 @@
 from easybuild.tools.ordereddict import OrderedDict
 from easybuild.tools.run import run_cmd
 from easybuild.tools.toolchain import DUMMY_TOOLCHAIN_NAME
-from easybuild.tools.utilities import quote_str
+from easybuild.tools.utilities import only_if_module_is_available, quote_str
 
 # optional Python packages, these might be missing
 # failing imports are just ignored
 # a NameError should be catched where these are used
 
-# PyGraph (used for generating dependency graphs)
-graph_errors = []
-try:
-    from pygraph.classes.digraph import digraph
-except ImportError, err:
-    graph_errors.append("Failed to import pygraph-core: try easy_install python-graph-core")
-
 try:
+    # PyGraph (used for generating dependency graphs)
+    # https://pypi.python.org/pypi/python-graph-core
+    import pygraph.classes.digraph as digraph
+    # https://pypi.python.org/pypi/python-graph-dot
     import pygraph.readwrite.dot as dot
-except ImportError, err:
-    graph_errors.append("Failed to import pygraph-dot: try easy_install python-graph-dot")
-
-# graphviz (used for creating dependency graph images)
-try:
+    # graphviz (used for creating dependency graph images)
     sys.path.append('..')
     sys.path.append('/usr/lib/graphviz/python/')
     sys.path.append('/usr/lib64/graphviz/python/')
```

Comment:
because it doesn't work without that, see also http://stackoverflow.com/questions/27302795/how-to-get-past-import-gv-in-python-graph